lib/subscribers/mysql/config.js

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Query subscriber configs
+const connectionQuery = {
```

Comment:
i would grouped these between `mysql` and `mysql2` above the bottom, it's very hard to see what's in each library